frontend/dockerfile/docs/reference.md

```diff
@@ -1556,11 +1556,14 @@ For more information about copying from named sources, see the
 
 #### Copying from the build context
 
-When copying source files from the build context, their paths are interpreted as
-relative to the root of the context. If you specify a relative path leading
-outside of the build context, such as `COPY ../something /something`, parent
-directory paths are stripped out automatically. The effective source path in
-this example becomes `COPY something /something`.
+When copying source files from the build context, paths are interpreted as
+relative to the root of the context.
+
+Specifying a source path with a leading slash or one that navigates outside the
+build context, such as `COPY ../something /something`, automatically removes
+any parent directory navigation (`../`). Trailing slashes in the source path
```

Comment:
In the example you gave, does `COPY ../something /something` get treated as `COPY something /something`? It's not immediately clear that this is the case...

Reply:
Yes, `COPY ../something /something` and `COPY something /something` are equivalent

Reply:
@dvdksn Any chance you could add this clarification to the updated document? Here is what the updated version would look like:

> Specifying a source path with a leading slash or one that navigates outside the
build context, such as `COPY ../something /something`, automatically removes
any parent directory navigation (`../`), making `COPY ../something/ /something` equivalent to `COPY
something /something`.